types/img-diff-js/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for img-diff-js 0.4
+// Project: https://github.com/reg-viz/img-diff-js#readme
+// Definitions by: Daniel Chif <https://github.com/nadchif>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface ImgDiffOptions {
+    actualFilename: string;
+    expectedFilename: string;
+    diffFilename?: string;
+    generateOnlyDiffFile?: boolean; // default false
+    options?: {
+        threshold?: number; // default 0.1
+        includeAA?: boolean; // default false
```

Comment:
Same for these comments.

```suggestion
        /** default: `0.1` **/
        threshold?: number;
        /** default: `false` **/
        includeAA?: boolean;
```